website/versioned_docs/version-0.60/parity-status.md

```diff
@@ -18,13 +18,13 @@ Below are a list of components and their status towards parity between the lean
 |`ActivityIndicator`|**Complete**|-|**Complete**|
 |`Button`|**Complete**|-|**Complete**|
 |`FlatList`|**Complete**|-|**Complete**|
-|`Image`|Partial|[`Image` project board](https://github.com/microsoft/react-native-windows/projects/18)|Partial|
-|`ScrollView`|Partial|[`ScrollView` project board](https://github.com/microsoft/react-native-windows/projects/17)|Partial|
+|`Image`|Partial|[`Image` project board](https://github.com/microsoft/react-native-windows/projects)|Partial|
+|`ScrollView`|Partial|[`ScrollView` project board](https://github.com/microsoft/react-native-windows/projects)|Partial|
 |`Switch`|**Complete**|-|**Complete**|
 |`Text`|**Complete** *(Some properties not supported due to device platform differences, docs coming soon)*|**Complete**|**Complete**|
-|`TextInput`|Partial|[`TextInput` project board](https://github.com/microsoft/react-native-windows/projects/20)|Partial|
+|`TextInput`|Partial|[`TextInput` project board](https://github.com/microsoft/react-native-windows/projects)|Partial|
 |`KeyboardAvoidingView`|**Complete**|-|Not supported|
-|`View`|Partial (some Style props left)|[`ViewStyle` props project board](https://github.com/microsoft/react-native-windows/projects/19)|**Complete**|
+|`View`|Partial (some Style props left)|[`ViewStyle` props project board](https://github.com/microsoft/react-native-windows/projects)|**Complete**|
 |`VirtualizedList`|Partial|No Issues Logged|**Complete**|
```

Comment:
Not necessary for this PR, but we probably want to rework this page.

Reply:
created a follow-up issue!